imports/plugins/included/product-admin/client/hocs/withVariantForm.js

```diff
@@ -189,6 +199,28 @@ const wrapComponent = (Comp) => (
       }
     }
 
+    updateVariantPrice = async (variantId, fieldName, value) => {
+      const { client, shopId } = this.props;
+      const [opaqueVariantId] = await getOpaqueIds([{ namespace: "Product", id: variantId }]);
+
+      try {
+        await client.mutate({
+          mutation: updateProductVariantPrices,
+          variables: {
+            input: {
+              shopId,
+              variantId: opaqueVariantId,
+              prices: {
+                [fieldName]: value
+              }
+            }
+          }
+        });
+      } catch (error) {
+        Alerts.toast(i18next.t("productDetailEdit.updateProductFieldFail", { [fieldName]: value }), "error");
```

Comment:
For new code we want to use the new notification library `notistack` as seen here: https://github.com/reactioncommerce/reaction-admin/blob/e9cc890e3a1b63cec0804b260a161ed60f685a79/imports/plugins/core/taxes/client/containers/GeneralTaxSettings.js#L31 Please update to use it.